tests/api/v1/samples/put.js

```diff
@@ -172,4 +172,42 @@ describe(`tests/api/v1/samples/put.js, PUT ${path} >`, () => {
       done();
     });
   });
+
+  describe('subject isPublished false >', () => {
+    beforeEach((done) => {
+      tu.db.Subject.findById(subjectId)
```

Comment:
is the "beforeEach" hook required? just a "before" hook will do.

Reply:
beforeEach is required, otherwise the "before" block to update the subject/aspect will try to run before the "beforeEach" in the outer test block, which creates the subject/aspect.
From official mocha documentation: https://mochajs.org/#hooks
  before(function() {
    // runs before all tests in this block
  });

  beforeEach(function() {
    // runs before each test in this block
  });